content/en/docs/instrumentation/cpp/exporters.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Exporters
+weight: 4
+---
+
+In order to visualize and analyze your
+[traces](/docs/concepts/signals/traces/#tracing-in-opentelemetry) and metrics,
+you will need to export them to a backend.
+
+## Logging exporter
+
+The logging exporter is useful for development and debugging tasks, and is the
+simplest to set up.
+
+```cpp
+auto ostream_exporter =
+    std::unique_ptr<opentelemetry::sdk::trace::SpanExporter>(new opentelemetry::exporter::trace::OStreamSpanExporter);
+```
+
+## OTLP endpoint
```

Comment:
below steps are only for OTLP HTTP Exporter, should we change the heading accordingly, or else change the steps below to indicate both HTTP and gRPC.

Reply:
added grpc as well